runtime/dart_vm.cc

```diff
@@ -326,6 +329,11 @@ DartVM::DartVM(const Settings& settings,
   // precompiled code only in the debug product mode.
   bool use_checked_mode = !settings.dart_non_checked_mode;
```

Comment:
Should we rename the variable use_checked_mode to use_asserts as checked mode used to mean something else in Dart 1

Reply:
Yeah, that makes sense. That should probably go in a separate PR.